### DISCUSSION TITLE
How to use analyze_logs.py for plotting AP data?

### DISCUSSION THREAD

**Discussion by seon-creator** [open-mmlab/mmdetection#10596] [Q&A]
Hello, I'm using mmdetection for project.
When I train the model, I can get  AP and recall value from log file.
I want to plot values by epochs.
How to use the file to plot the data?
If another method exists, tell me how to plot data from the log file.

I attached the log file and analyze_logs.py

# mmdetection/tools/analysis_tools/analyze_logs.py
analyze_logs.py is in the mmdetection folder.

[20230705_114548.log](https://github.com/open-mmlab/mmdetection/files/11954449/20230705_114548.log)

**Comment by ghlerrix** [comment]
You can draw using a json file with the same name as the log file. The details can be seen: [https://mmdetection.readthedocs.io/en/latest/user_guides/useful_tools.html#log-analysis](https://mmdetection.readthedocs.io/en/latest/user_guides/useful_tools.html#log-analysis)
